src/Microsoft.DotNet.Tools.Init/Template/project.json

```diff
@@ -6,7 +6,7 @@
 
     "dependencies": {
         "Microsoft.NETCore.Runtime": "1.0.1-beta-*",
-        "System.IO": "4.0.10-beta-*",
+        "System.IO": "4.0.11-beta-*",
```

Comment:
Temporary issue. We need to deal with this until Standard Platform comes online.